components/post_view/reaction_list/reaction_list.jsx

```diff
@@ -57,10 +57,11 @@ export default class ReactionList extends React.PureComponent {
         this.state = {
             showEmojiPicker: false,
         };
+        this.addReactionButton = React.createRef();
```

Comment:
thank you for the suggestion @Willyfrog i have committed next time i will make it like that also

Reply:
also @Willyfrog i don't understand about the ```reference in the rendered component. ``` because the rendered component before is using method to get the ```addReactionButton``` like ```this.getTarget``` do i need to change all of this into this.addReactionButtonRef.current instead?

Reply:
I'll try to clarifiy:
in this line: https://github.com/mattermost/mattermost-webapp/pull/6647/files#diff-17f9bf4c8b32d180a9d536f1ae4b0725R163 
the reference is set as a string reference, we need to replace it with the created one.